src/components/DocumentDescription/index.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import documentDescriptionStyles from './styles';
+import { useSelector } from '../../reducers/rootReducer';
+import messages from './messages';
+
+interface DocumentDescriptionProps {
+  className?: string,
+  documentId?: string,
+}
+
+const getDocumentContent = (documentType: string | null) => {
+    switch (documentType) {
+        case "1":
```

Comment:
maybe we can use constants instead

Reply:
Just use the object Keys instead a switch case